sql/2019/04_Media/04_01.sql

```diff
@@ -0,0 +1,128 @@
+#standardSQL
+/*
+04_01
+04_02
+
+1.22 TB query
+
+this query takes lighthouse data on image performance: 
+    lazyloading (offscreen), 
+    responsiveness, 
+    quality (optimized),
+    format (webp),
+    animated GIF
+    
+    I break down this data into: 
+         scores (from 0-1) 
+         wasted ms in load time
+         wasted KB
+         count of offending items
+    and break them into percentiles - 26 is the median, but 50 seemed 
```

Comment:
It looks like PWA is not calculated as a test yet.

I'll get more percentiiles for the graphs..
..and I suppose I'm used to trying to be super efficient and combining my queries to stay in my budget :)  I'll strip these into multiple queries.  How many separate should I create?  I have 4-5 scores, 4-5 millisecond savings, 4-5 KB savings, 4-5 "count" the number of failures per site.

Should I do all the scores as one (all ms as one, etc)?  thats 4 queries.. If I do each uniquely - thats like 25 queries

Reply:
new pull request will have 8 queries... 4 with 1000 percentile points to graph the scores...
then 4 to give the 0, 25, 50 75 and 100 percentile savings for websites that score <0.5 on the lighthouse test

Reply:
Ah ok thanks for explaining. I've added a comment to https://github.com/HTTPArchive/almanac.httparchive.org/issues/67#issuecomment-517350413 with a note for the developers to expect to handle multiple metrics in a single query/JSON result.

As long as your field names are aliased descriptively we should have the ability to do something like "generate a histogram based on field `webpscore` in `04_01`".

Since the 04_01[a-z] queries have so much redundancy I agree with your original approach that they can be combined into one query. Sorry for the extra work, but your way makes more sense.

Reply:
I have pulled 0402_b and 0401_b and 0401_c back into 0401a.

The e-g are different enough that they should be different queries.